# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/screen/glibc23.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./extern.h.orig	2003-03-12 17:57:42.000000000 +0100
+++ ./extern.h	2003-03-12 17:58:09.000000000 +0100
@@ -344,9 +344,9 @@
 #if defined(POSIX) || defined(hpux)
 extern void (*xsignal __P((int, void (*)SIGPROTOARG))) __P(SIGPROTOARG);
 #endif
-#ifndef HAVE_RENAME
-extern int   rename __P((char *, char *));
-#endif
+// #ifndef HAVE_RENAME
+// extern int   rename __P((char *, char *));
+// #endif
 #if defined(HAVE_SETEUID) || defined(HAVE_SETREUID)
 extern void  xseteuid  __P((int));
 extern void  xsetegid  __P((int));
--- ./os.h.orig	2003-03-12 18:00:22.000000000 +0100
+++ ./os.h	2003-03-12 18:01:00.000000000 +0100
@@ -172,9 +172,9 @@
 #ifndef HAVE_UTIMES
 # define utimes utime
 #endif
-#ifndef HAVE_VSNPRINTF
-# define vsnprintf xvsnprintf
-#endif
+// #ifndef HAVE_VSNPRINTF
+// # define vsnprintf xvsnprintf
+// #endif
 
 #ifdef BUILTIN_TELNET
 # include <netinet/in.h>
--- ./misc.c.orig	2003-03-12 17:58:44.000000000 +0100
+++ ./misc.c	2003-03-12 17:59:45.000000000 +0100
@@ -76,7 +76,7 @@
 int err;
 {
   extern int sys_nerr;
-  extern char *sys_errlist[];
+//extern char *sys_errlist[];
 
   static char er[20];
   if (err > 0 && err < sys_nerr)
@@ -423,6 +423,7 @@
 #endif
 }
 
+#if 0
 #ifndef HAVE_RENAME
 int
 rename (old, new)
@@ -434,6 +435,7 @@
   return unlink(old);
 }
 #endif
+#endif
 
 
 int
